test_collections/matter/sdk_tests/support/yaml_tests/models/chip_test.py

```diff
@@ -170,35 +174,77 @@ def step_failure(
     def step_unknown(self) -> None:
         self.__runned += 1
 
-    async def step_manual(self) -> None:
+    async def step_manual(self, request: TestStep) -> None:
```

Comment:
The `hooks.py` file uses the "request" as optional. We should probably override the same way to maintain it compatible.
```suggestion
    async def step_manual(self, request: Optional[TestStep]) -> None:
```